service/cloudconfig/cloud_config.go

```diff
@@ -55,13 +56,13 @@ func (c *CloudConfig) NewMasterTemplate(customObject awstpr.CustomObject, certs
 	var err error
 
 	// TODO remove defaulting as soon as custom objects are configured.
-	if customObject.Spec.Cluster.Version == "" {
+	if key.ClusterVersion(customObject) == "" {
```

Comment:
I updated the cloudconfig service to use the new key helper.

Reply:
You can use `HasClusterVersion` here right`?

Reply:
Good point, done.